index.html

```diff
@@ -114,7 +115,7 @@ <h3>
                 <hr class="underline desktop">
             </section>
 
-            <section id="featured-speakers"></section>
+            <section id="featured-stars"></section>
```

Comment:
Great use of semantics 😎 The proper use of tags is key to having better HTML! Well done using section and footer tags 💪 We must not forget to use the `article` tag! Kindly make the necessary changes in your speaker's HTML dynamic creation so that each of them has an `article` tag 🤓